src/main/java/com/tradeshift/HelloWorld.java

```diff
@@ -0,0 +1,47 @@
+package com.tradeshift;
+
+import java.net.HttpURLConnection;
+import javax.ws.rs.GET;
+import javax.ws.rs.Path;
+import javax.ws.rs.Produces;
+import javax.ws.rs.QueryParam;
+import javax.ws.rs.WebApplicationException;
+import javax.ws.rs.core.Response;
+
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.*;
+
+import com.tradeshift.service.HelloWorldInterface;
+
+
+@Path("/")
+@Controller
+public class  HelloWorld {
+
+    @Autowired
+    private HelloWorldInterface helloWorldService;
```

Comment:
To help with testing later, it's best practice to not do private field injection, but instead do constructor injection. This can be achieved by rewriting this into:
```java
private HelloWorldInterface helloWorldService;

public HelloWorld(HelloWorldInterface helloWorld) {
  this.helloWorldService = helloWorld;
}
```
This makes sure that you can inject test classes manually from tests if/when needed.